component/src/main/java/org/wso2/extension/siddhi/io/http/sink/util/HttpSinkUtil.java

```diff
@@ -0,0 +1,196 @@
+/*
+ *  Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.extension.siddhi.io.http.sink.util;
+
+import org.apache.log4j.Logger;
+import org.wso2.carbon.messaging.Header;
+import org.wso2.carbon.transport.http.netty.config.SenderConfiguration;
+import org.wso2.carbon.transport.http.netty.config.TransportProperty;
+import org.wso2.extension.siddhi.io.http.sink.exception.HttpSinkAdaptorRuntimeException;
+import org.wso2.extension.siddhi.io.http.util.HttpConstants;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * {@code HttpSinkUtil } responsible of the all configuration reading and input formatting of http transport.
+ */
+public class HttpSinkUtil {
+    private static final Logger log = Logger.getLogger(HttpSinkUtil.class);
+
+    public HttpSinkUtil() {
```

Comment:
What is the usage of his empty public constructor?

Reply:
Fixed in 156b93ab839dfc225e0a915f8951b61db7afa2ab